scripts/deploy.sh

```
@@ -0,0 +1,159 @@
+
+ADMIN_ADDRESS=$(stellar keys address $ADMIN_ALIAS)
+FEE_WALLET_ADDRESS=$(stellar keys address $FEE_WALLET_ALIAS)
+
+# Handle XLM SAC
+echo ""
+echo "🔍 Resolving XLM SAC..."
+if [ "$NETWORK" == "testnet" ]; then
+    XLM_SAC="CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC"
+else
+    # For local, we might need to deploy it if it doesn't exist
+    stellar contract asset deploy --asset native --source $ADMIN_ALIAS --network $NETWORK > /dev/null 2>&1 || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)